src/siaskydev.test.ts

```diff
@@ -29,4 +31,48 @@ describe.skip("siasky.dev end to end", () => {
     const buf = Buffer.from(parts[1], "base64");
     expect(buf.toString("ascii")).toEqual("thisistext");
   });
+
+  it("support rev number update higher than 255", async () => {
+    // create a random user
+    const rand = randomNumber(0, 1e6);
+    const user = new User(`john.doe+${rand}@gmail.com`, "test1234");
+    console.log(`Uploading for userid ${user.id}`);
+
+    // verify there is no entry
+    let existing = null;
+    try {
+      existing = await promiseTimeout(client.lookupRegistry(user, fileID), 1e4);
+    } catch (error) {
+      // expected
+    }
+    expect(existing).toBeNull();
+
+    // build the registry value
+    let revision = 0;
+    const skylink = "CABAB_1Dt0FJsxqsu_J4TodNCbCGvtFf1Uys_3EgzOlTcg";
+
+    // update the registry 256 times
+    for (revision; revision <= 260; revision += 10) {
+      if (revision && revision % 50 === 0) {
+        console.log(`Iteration ${revision}...`);
```

Comment:
remove console.log before merging that to master

Reply:
Why? The `siaskydev.test.ts` file is skipped and marked as follows:

```
// skip - used to verify end-to-end flow
describe.skip("siasky.dev end to end", () => {
```

Reply:
I'll resolve all the `console.log` remarks and keep this open.